<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_exist_group</class>
 <widget class="QDialog" name="edit_exist_group">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>edit_exist_group</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>group name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTextEdit" name="subInfoEdit">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>user define</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QListWidget" name="listWidget">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="positionBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>localization</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="repositionBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>deviation</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QCheckBox" name="systemLogBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>system_log</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>89</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="deleteButton">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">.QPushButton {
	border-style: none;
	border: 0px;
	min-height:32px;
    min-width:70px;
	border-radius: 8px;
	color: rgb(255, 255, 255);
	
	background-color: rgb(26, 188, 156);
    background-repeat:no-repeat;
}


.QPushButton:hover{ 
background-color: rgb(74, 215, 187);
    background-repeat:no-repeat;
}

.QPushButton:pressed{ 
background-color: rgb(19, 159, 131);
}
</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="okButton">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">.QPushButton {
	border-style: none;
	border: 0px;
	min-height:32px;
    min-width:70px;
	border-radius: 8px;
	color: rgb(255, 255, 255);
	
	background-color: rgb(26, 188, 156);
    background-repeat:no-repeat;
}


.QPushButton:hover{ 
background-color: rgb(74, 215, 187);
    background-repeat:no-repeat;
}

.QPushButton:pressed{ 
background-color: rgb(19, 159, 131);
}
</string>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="cancelButton">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">.QPushButton {
	border-style: none;
	border: 0px;
	min-height:32px;
    min-width:70px;
	border-radius: 8px;
	color: rgb(255, 255, 255);
	
	background-color: rgb(26, 137, 188);
    background-repeat:no-repeat;
}


.QPushButton:hover{ 
background-color: rgb(88, 174, 214);
    background-repeat:no-repeat;
}

.QPushButton:pressed{ 
background-color: rgb(20, 113, 157);
}
</string>
     </property>
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>edit_exist_group</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>edit_exist_group</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
